rules/anti-static/obfuscation/python.yara

```diff
@@ -449,10 +449,10 @@ rule decompress_base64_entropy: high {
     $f_decompress = "decompress("
     $f_eval       = "eval("
     $f_exec       = "exec("
-    $long_str     = /[\'\"][\+\w\/]{96}/
+    $long_str     = /[\'\"][\+\w\/]{,96}[\'\"]/
```

Comment:
We don't have a test for this, but this rule will no longer work as expected: the intent of this rule is to find a base64-encoded string of at least 96 characters. I played with length ranges like {96,8192}, but it slowed the query down even more than just saying {96} and removing the quotes at the end.

Reply:
Ah okay, that makes sense.

Reply:
I still get a compiler warning with the changes in [`5ed3568` (#708)](https://github.com/chainguard-dev/malcontent/pull/708/commits/5ed3568d0e92415164899275755d13ea63785143) but it's much faster.